.travis.yml

```diff
@@ -159,20 +207,24 @@ script:
   # - export PYTHONPATH="$PYTHONPATH:./ci/"
 
   # ray tune tests
+  - if [ $RAY_CI_TUNE_AFFECTED == "1" ]; then exit; fi
```

Comment:
we don't want to exit here because if `tune` is not affected, but something else is (e.g., rllib), then we still need to run the rllib tests

instead, maybe do 

```
- if [ $RAY_CI_TUNE_AFFECTED == "1" ]; then ./ci/suppress_output python python/ray/tune/tests/test_dependency.py; fi
```

and similarly for the next line.

It's a bit verbose, but that might be the best..